src/jquery.eclipsefdn-api.js

```diff
@@ -1991,6 +1991,21 @@
         // make sure we have a promotion to display
         if (json.length > 0) {
           writeFeaturedContainer(json[0], $container, type);
+        } 
```

Comment:
bit of a nitpick, but we should just use `else` rather than defining another if block